.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -147,6 +147,9 @@ NVIDIA-GRACE-GRACE:
   image: ubuntu:latest
   script:
     - apt-get update && apt-get install -y cmake git g++
+    - export OMP_NUM_THREADS=8
```

Comment:
We could likely use more threads but I'm not sure how much we want to fiddle with it given that this is already much better.

Reply:
Considering that the system has 144 cores, I think we should increase that number.
What about 72?

Reply:
With 72 we're back at this test going long, 
```
[       OK ] openmp.team_reduction_scan (142995 ms)
```

Reply:
This is because you have too much contentions on the small loops. IIRC you want to use 12 or less.

Reply:
Just for this test, seems like less is faster. 4 faster than 8 for example.